codecs/libdvdnav.json

```diff
@@ -4,8 +4,8 @@
         {
             "type": "git",
             "url": "https://code.videolan.org/videolan/libdvdnav.git",
-            "tag": "6.0.0",
-            "commit": "dcb9109e45ccd304ec82a7c7bf46cca63620adf9"
+            "tag": "6.1.0",
+            "commit": "4f48efd43efb2e3372cb494a8893342e1fb507ae"
```

Comment:
Please add justification in the commit message for each of the upgrades (eg. "Fix crashes on some DVDs").